tests/scripts/istio-iptables-test.sh

```diff
@@ -57,15 +57,9 @@ function compareWithGolden() {
   local FILE_UNDER_TEST
 
   case "${TEST_MODE}" in
-   "script")
-    FILE_UNDER_TEST="${SCRIPT_DIR}/../../tools/packaging/common/istio-iptables.sh"
-   ;;
    "golang")
```

Comment:
Should we just make this a proper golang test? It probably should have been this way from the start and just shelled out to the shell script, unless there is some reason it should be like this? Could be a follow up pr

Reply:
yeah. we can fix this in a separate PR